index.html

```diff
@@ -18,6 +23,23 @@
         <li><img src="./assets/envelope.png" alt="mail_icon"/></li>
       </ul>
       <img id="hamburger_menu" class="menu" src="./assets/Menu.png" alt="menu"/>
+      <div id="menu_mobile" class="menu_mobile">
+        <div id="close_button" class="img">
+          <img src="./assets/Enabled_button.png" alt="X-button">
+        </div>
+        <div class="portfolio_container">
+          <span><a href="#portfolio">Portfolio</a></span>
+          <img src="./assets/arror_right.png" alt="arrow">
+        </div>
+        <div class="about_container">
+          <span><a href="#About">About</a></span>
+          <img src="./assets/arror_right.png" alt="arrow">
+        </div>
+        <div class="contact_container">
+          <span><a href="#contact">Contact</a></span>
+          <img src="./assets/arror_right.png" alt="arrow">
+        </div>
+      </div>
     </nav>
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT**: Good job so far with your mobile menu. It is working as expected and has good styling. So, I noticed that you implemented two **navigations lists** (portfolio, Contact, and About) and they only work on the mobile menu(as on Desktop they should behave in the way if you click on one of the options it should lead you to the correct section as for mobile). 
To make your project complete and be professional, you are supposed to use one navigation list and implement the javascript logic so it works both on desktop and mobile menu. i.e if you are on the Desktop or Mobile and you click on **portfolio** or **Contact** or **About** you should be directed to the correct section.
You can do it like this:
```bash
- First, you remove the 1st navigation list and edit the second one to be like in the code below 
- Make sure you style them as  from the Figma design
- Then, add the JavaScript logic to make them work as you did. Using the code below or similar to this,  it will help you to fix the issue in the Desktop version.

<header>
  <a class="logo" href="#">Davis Nyale</a>
  <nav class="navbar">
    <img id="hamburger_menu" class="menu" src="./assets/Menu.png" alt="menu"/>
    <ul id="menu_mobile" class="menu_mobile">
      <li id="close_button" class="img">
        <img src="./assets/Enabled_button.png" alt="X-button">
      </li>
      <li class="portfolio_container">
        <span><a href="#portfolio">Portfolio</a></span>
        <img src="./assets/arror_right.png" alt="arrow">
      </li>
      <li class="about_container">
        <span><a href="#About">About</a></span>
        <img src="./assets/arror_right.png" alt="arrow">
      </li>
      <li class="contact_container">
        <span><a href="#contact">Contact</a></span>
        <img src="./assets/arror_right.png" alt="arrow">
      </li>
      <li>
        <img src="./assets/envelope.png" alt="mail_icon"/>
      </li>
    </ul>
  </nav>
</header>
```
#### Please make sure you implement this feature as it was supposed to work before you start the mobile menu and this is why they didn't include it again in the project's requirements(but it should be required).